core/src/error.rs

```diff
@@ -37,8 +36,8 @@ pub enum DefuseError {
     #[error("JSON: {0}")]
     JSON(#[from] serde_json::Error),
 
-    #[error("NFT '{}' was already deposited", TokenId::Nep171(.0.clone(), .1.clone()))]
-    NftAlreadyDeposited(AccountId, defuse_nep245::TokenId),
+    #[error("NFT '{0}' is already deposited")]
```

Comment:
This will not include `nep171:` prefix, so maybe wrap in `TokenId`?
```suggestion
    #[error("NFT '{}' is already deposited", TokenId::Nep171(.0.clone()))]
```